.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: python
-python:
-  - 2.7
-  - 3.3
-  - 3.4
+env:
+  - TOXENV=py26
```

Comment:
Can you add a comment above the list of environments noting that this is a duplicate of what is in `tox.ini`? Something like this:

```yaml
env:
  # Should be identical to envlist in tox.ini. See:
  # http://tox.readthedocs.org/en/latest/config.html#confval-envlist=CSV
  - TOXENV=py26
  - TOXENV=py27
  - TOXENV=py33
  - TOXENV=py34
```